src/agent/structures/structures.ts

```diff
@@ -88,8 +95,9 @@ export interface ResumeChatParameters {
 }
 
 export interface TransferChatParameters {
+  id: string;
   target?: TransferTargetGroup | TransferTargetAgent;
-  force?: boolean;
+  force: boolean;
```

Comment:
```suggestion
  force?: boolean;
```
It is optional according to the [docs](https://developers.livechat.com/docs/messaging/agent-chat-api/v3.3/rtm-reference#transfer-chat) and the golang default value for bool is false (like the docs say)

Reply:
OK in v3.5, 3.4, 3.6
fixed in 3.3